classes/mail/mailables/ReviewerResendRequest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file classes/mail/mailables/ReviewerResendRequest.php
+ *
+ * Copyright (c) 2014-2022 Simon Fraser University
+ * Copyright (c) 2000-2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class ReviewerResendRequest
+ * @ingroup mail_mailables
+ *
+ * @brief   Email sent to a review who has declined their review assignment to ask them
+ *          to reconsider accepting the review assignment
+ */
+
+namespace PKP\mail\mailables;
+
+use PKP\context\Context;
+use PKP\mail\traits\Configurable;
+use PKP\mail\Mailable;
+use PKP\mail\traits\Recipient;
+use PKP\mail\traits\Sender;
+use PKP\security\Role;
+use PKP\submission\PKPSubmission;
+use PKP\submission\reviewAssignment\ReviewAssignment;
+
+class ReviewerResendRequest extends Mailable
+{
+    use Recipient;
+    use Sender;
+    use Configurable;
+
+    protected static ?string $name = 'mailable.reviewerResendRequest.name';
+    protected static ?string $description = 'mailable.reviewerResendRequest.description';
+    protected static ?string $emailTemplateKey = 'REVIEW_RESEND_REQUEST';
+    protected static bool $supportsTemplates = true;
+    protected static array $groupIds = [self::GROUP_REVIEW];
+    protected static bool $canDisable = true;
+    protected static array $fromRoleIds = [Role::ROLE_ID_MANAGER];
+    protected static array $toRoleIds = [Role::ROLE_ID_REVIEWER];
+
+    public function __construct(Context $context, PKPSubmission $submission, ReviewAssignment $reviewAssignment)
```

Comment:
Oh, this is because of this code:

https://github.com/pkp/pkp-lib/blob/main/controllers/grid/users/reviewer/form/ReviewerNotifyActionForm.php#L74-L87

Vitaliy is in the process of removing the last bits of the `SubmissionMailTemplate` and this will probably get fixed when that is done. Can you file an issue to solve this later?

Reply:
@NateWr issue filed at https://github.com/pkp/pkp-lib/issues/8262 .